canvas/fizz-27-py/readme.md

```diff
@@ -0,0 +1,29 @@
+# Getting data into CSV from Canvas API
+
+### Canvas REST API
+
+This script gets the data from the Canvas REST API
+[Canvas docs](https://canvas.instructure.com/doc/api/)
+
+### What does it do?
+
+Generates some CSV files with information for a specified year, extracted from Canvas REST API:
+
+* Courses
+* Assignments
+* Students
+* Enrollments
+* Submissions
+
+### Requirements
+
+* python 3.8
+* pip 20.2.2
+* pipenv 2020.8.13
```

Comment:
We should all discuss moving from pipenv to Poetry. It was an easy move and has been a much better experience for me. It also sets the stage for easy publication to Azure Artifacts.